blueprints/ember-electron/files/ember-electron/main.js

```diff
@@ -32,6 +45,9 @@ app.on('ready', () => {
   mainWindow = new BrowserWindow({
     width: 800,
     height: 600,
+    webPreferences: {
+      nodeIntegration: true,
```

Comment:
It would be nice not to require `nodeIntegration` to be enabled in order to work. See https://github.com/felixrieseberg/ember-electron/pull/390#issuecomment-486440096